deel-2-flexbox-layout/opdracht-2/styles/styles.css

```diff
@@ -1 +1,39 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+/*Schrijf hier jouw CSS*/
+body {
+    display: flex;
+    flex-direction: row;
+    min-height: 100vh;
+}
+
+div , main {
```

Comment:
Hier doe jij de div en main met elkaar combineren. Toen ik keek naar het resultaat zag ik dat de grootte van article ver over mijn scherm heen ging. Main heb ik toen verwijderd omdat ik niet kon zien wat dit deed en dat veranderde ook niets.

Toen ik ging zoeken in de uitwerkingen en met een beetje puzzelen kwam ik tot deze uitkomst die beter werkt:
Bij regel 8 ', main' verwijderen en daarvan maken 'div {'

Daarna op een lege regel eronder:
main{
    display: flex;
    flex-direction: row;
    flex-basis: auto;
    flex-grow: 1;
    flex-shrink: 1;
}

Hierdoor lijnt hij mooi uit op het volledige scherm. 

Ook merkte ik dat ik goed moest kijken welke <div> je hier bedoelde. Natuurlijk zit er maar 1 div in dit bestand maar ik kan me voorstellen dat bij grotere sites dit moeilijk zoeken kan zijn. Ik heb geloof ik zelf de al door Novi aangemaakt classe : col-2 gebruikt. Die kon ik sneller opzoeken.